src/core/LiterallyCanvas.coffee

```diff
@@ -313,7 +320,7 @@ module.exports = class LiterallyCanvas
     return unless shapes.length
     drawShapes = =>
       for shape in shapes
-        renderShapeToContext(ctx, shape, {retryCallback})
+        shape.draw(ctx, { retryCallback })
```

Comment:
(Will switch it back; also shape.drawLatest in the shapes in progress loop I assume)